ts/mixpanel.ts

```diff
@@ -16,22 +15,22 @@ export const initializeMixPanel = async (state: GlobalState) => {
   if (mixpanel !== undefined) {
     return;
   }
-  const privateInstance = new MixpanelInstance(mixpanelToken);
-  await privateInstance.initialize();
+  const trackAutomaticEvents = false;
+  const privateInstance = new Mixpanel(mixpanelToken, trackAutomaticEvents);
+  await privateInstance.init(
+    undefined,
+    undefined,
+    "https://api-eu.mixpanel.com/"
```

Comment:
```suggestion
    "https://api-eu.mixpanel.com"
```
I've found that with the trailing slash we have a `301` status code from mixpanel.

![image](https://github.com/pagopa/io-app/assets/16268789/34f6a9d9-9b2d-4bfa-9f09-040b38645c58)

This lead to a missing event tracking.